lib/rialto/etl/namespaces.rb

```diff
@@ -2,6 +2,15 @@
 
 require 'rdf'
 
+module RDF
+  # Adds method to Vocabulary extracting id from URI for this vocabulary
+  class Vocabulary
+    def remove_from_uri(uri)
```

Comment:
I'm not sure this should be added to `RDF::Vocabulary` if there are alternatives. That is a class from a dependency, it's not our code.

Reply:
Open to suggestions on most Rubyish way to implement.

Reply:
Add it to a module in the `Rialto::ETL` namespace and `include` it into the relevant vocabularies.